sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -135,13 +135,11 @@ public:
   bfloat16(const float &a);
   bfloat16 &operator=(const float &a);
 
-  // Convert from bfloat16 to float
+  // Convert bfloat16 to floating-point types
   operator float() const;
+  operator sycl::half() const;
 
-  // Get bfloat16 as uint16.
-  operator storage_t() const;
-
-  // Convert to bool type
+  // Convert bfloat16 to bool type
   explicit operator bool();
 
   friend bfloat16 operator-(bfloat16 &bf) { /* ... */ }
```

Comment:
Any changes to bfloat16 support can be taken up as later PRs.

Reply:
Tagging @rolandschulz for comment. Roland, can you please take a look at Greg's comment here? Thanks

Reply:
The access modifier doesn't matter for a friend declaration. This already is a hidden friend function. Because it is defined inside the class rather then at namespace scope. And therefore is only found by ADL and therefore isn't found if the argument would need to be implicitly converted first.

Reply:
Thanks, @rolandschulz .  I was thinking that the access specifier made a difference here, but I guess that is not the case.